docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -657,7 +685,7 @@ E.g.:
 - `0x5ef83ad9559033e6e941db7d7c495acd` (**length: 16 bytes** = `0x0010`)
 - `0xbeefbeef` (**length: 4 bytes** = `0x0004`)
 
-A CompactBytesArray for these 3 ERC725Y Data Keys would look like this:
+A CompactBytesArray for these 3 x ERC725Y Data Keys would look like this:
```

Comment:
```suggestion
A CompactBytesArray for these 3 different ERC725Y Data Keys would look like this:
```

Might be worth explaining that they are 3 specific keys? What was the point of the `x` besides stating the number of keys?